db/schema.rb

```diff
@@ -87,8 +87,8 @@
     t.bigint "project_id"
```

Comment:
why schema is changed without any migrations? remove it

Reply:
removed